### ISSUE TITLE
Logging seems to have garbage for duration

### ISSUE THREAD

**Issue by ankon** [jtblin/kube2iam#168]
With 0.10.3:
~~~~
kube2iam-996vs kube2iam time="2018-09-05T13:58:58Z" level=info msg="GET /latest/meta-data/iam/security-credentials/mate (200) took %!d(float64=160980) ns" req.method=GET req.path=/latest/meta-data/iam/security-credentials/mate req.remote=10.2.83.8 res.duration=160980 res.status=200
~~~~

With 0.9.0 I can see the duration entries correctly:
~~~~
kube2iam-tnvgx kube2iam time="2018-09-05T14:00:26Z" level=info msg="GET /latest/meta-data/iam/security-credentials/ (200) took 29059 ns" req.method=GET req.path=/latest/meta-data/iam/security-credentials/ req.remote=10.2.16.22 res.duration=29059 res.status=200
~~~~

**Comment by ankon** [comment]
I'm somewhat suspecting that this popped up in the Prometheus magic: https://github.com/jtblin/kube2iam/commit/ff7db7da4b678fac5e6fe51ec8801955c0990a02#diff-91bbeda7eb98a7adc57b9e47e2cf5c2bL115

Guessing: `Duration.Nanoseconds()` returns an int64, while we now have a float64? My go isn't strong enough here ... :)

**Comment by jrnt30** [comment]
Yes, https://github.com/jtblin/kube2iam/pull/167 should address this.

**Comment by jrnt30** [comment]
Closed via #167
